### DISCUSSION TITLE
Feature request: Ability to measure objects

### DISCUSSION THREAD

**Discussion by alexjnoble** [nextpyp/.github#12] [Ideas]
Hello,

I would like to make a feature request: Give the user a method for measuring the size and shape of objects from their web browser. ChatGPT suggests to use HTML Canvas or JavaScript libraries such as fabric.js, Konva.js, or paper.js to accomplish this.

This feature would allow users to quickly decide what parameters to use when auto-picking in nextPYP.

Thanks,
-Alex

**Comment by abartesaghi** [comment]
Alex, thanks for the suggestion, we can certainly implement this.

**Reply by abartesaghi** [reply]
This feature has been implemented in [v0.5.3](https://nextpyp.app/files/pyp/0.5.3/docs/index.html)
